bang/forms.py

```diff
@@ -560,6 +560,9 @@ def _gacha_type_to_queryset(self, queryset, request, value):
     ])
     gacha_type_filter = MagiFilter(to_queryset=_gacha_type_to_queryset)
 
+    featured_member = forms.ChoiceField(choices=BLANK_CHOICE_DASH + [(id, full_name) for (id, full_name, image) in getattr(django_settings, 'FAVORITE_CHARACTERS', [])], initial=None, label=_('Featured Member'))
```

Comment:
Can you make the label just "Member"? To avoid adding translations terms to POEditor 😅

Reply:
✅